tools/build.js

```diff
@@ -15,7 +15,7 @@ const libRoot = path.join(__dirname, '../lib');
 const distRoot = path.join(libRoot, 'dist');
 const esRoot = path.join(libRoot, 'es');
 
-const clean = () => fse.existsSync(libRoot) && fse.remove(libRoot);
+const clean = async () => fse.existsSync(libRoot) && fse.remove(libRoot);
```

Comment:
just don't await this below; this isn't `async`

Reply:
`remove` is

Reply:
ah shoot, we didn't merge https://github.com/react-bootstrap/react-bootstrap/pull/3501

Reply:
okay, can undo changes here now that we've merged

Reply:
Done.